easybuild/framework/easyblock.py

```diff
@@ -337,6 +341,59 @@ def get_checksum_for(self, checksums, filename=None, index=None):
         else:
             raise EasyBuildError("Invalid type for checksums (%s), should be list, tuple or None.", type(checksums))
 
+    def fetch_source(self, source=None, checksum=None, extension=False):
```

Comment:
This should just be ` def fetch_source(self, source, ...):`, it doesn't make sense to call `fetch_source` without passing a value for `source`